src/eba_handlers.eliom

```diff
@@ -94,6 +94,12 @@ let send_act email userid =
   Lwt.return ()
 
 let sign_up_handler' () email =
+  let send_act email userid =
+    let msg =
+      String.concat ", " ["Welcome!\r\nTo confirm your e-mail address";
+                          "please click on this link: "] in
```

Comment:
I just tried it, however it seems that I wouldn't have
the "please"
written in the same column as in "Welcome", 
using multi-line strings

let msg = "Welcome!\r\nTo confirm your e-mail address \
 please click on this link: ";;
val msg : string =
  "Welcome!\r\nTo confirm your e-mail address please click on this link"
# let msg = "Welcome!\r\nTo confirm your e-mail address \ 
                    please click on this link: ";;
val msg : string =
  "Welcome!\r\nTo confirm your e-mail address  \n           please click on this link: "